README.md

```diff
@@ -30,25 +30,20 @@ markdown("minimal http://example.org/", extensions=["mdx_linkify"])
 It's possible to omit links that match your custom filter with linkify
 callbacks.
 
-For example, to omit links that end with `.txt` extension:
+For example, to omit links that end with `.net` extension:
 
 ```python
 from mdx_linkify.mdx_linkify import LinkifyExtension
 from markdown import Markdown
 
-def dont_linkify_txt_extension(attrs, new=False):
-    if attrs["_text"].endswith(".txt"):
+def dont_linkify_net_extension(attrs, new=False):
+    if attrs["_text"].endswith(".net"):
         return None
 
     return attrs
 
 md = Markdown(
-    extensions=[LinkifyExtension()],
-    extension_configs={
-        "linkify": {
-            "linkify_callbacks": [[dont_linkify_txt_extension], ""]
-        }
-    }
+    extensions=[LinkifyExtension(linkify_callbacks=[dont_linkify_txt_extension])],
```

Comment:
The preferred method is to pass in an instance of an extension: https://python-markdown.github.io/reference/#extension_configs